### PR TITLE
Add a version of TOTG `computeTimeStamps()` for combining sub-trajectories

### PR DESCRIPTION
**Pull request by AndyZe**
### Description

If the user has planned several sub-paths or trajectories, they often want to combine them and re-parameterize so the motion is smooth.

### CODE REVIEW

**Comment by simonschmeisser** [comment]
It would be lovely to have a variant where you can specify max velocity per segment instead of for the whole combined trajectory.

**Comment by AndyZe** [comment]
@simonschmeisser I think that would be tricky. TOTG is about blending the radii of circles around the waypoints, so I don't know what would happen if you suddenly switched the circle radius.

I'm sure there's a way but that would be way deeper than this PR.

File: moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp [hunk 1]
```diff
@@ -1026,4 +1026,22 @@ bool TimeOptimalTrajectoryGeneration::computeTimeStamps(robot_trajectory::RobotT
 
   return true;
 }
+
+bool TimeOptimalTrajectoryGeneration::computeTimeStamps(
```
Comment (henningkayser) [inline moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp:1030]: This function doesn't really look specific to TOTG at all. Why not just append the trajectories first and call the existing one?

Comment (AndyZe) [reply moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp:1030]: That's a good point. Assuming people can find the API for that. I'll just close this.

**Pull request closed without merging**